.cirrus.yml

```diff
@@ -166,11 +161,14 @@ task:
 
   build_image_script: |
     DATE=$(date --utc +'%Y-%m-%dt%H-%M-%S')
+    PKRVARFILE=packer/bsd/${CIRRUS_TASK_NAME%-*}.pkrvars.hcl
     export PACKER_LOG=1
 
     packer build \
+      -only "${BUILD_TYPE}.*" \
       -timestamp-ui \
       -force \
+      -var-file="${PKRVARFILE}" \
```

Comment:
Could you adjust the top-level Makefile as well? I use that to locally test that packer file is at least basically sound, with `make pre-commit`.